_includes/common-contact.html

```diff
@@ -1,25 +1,25 @@
 <section id="contact" class="container-fluid bg-light p-md-5">
     <div class="row  justify-content-around">
         <div class="col col-md-8 text-center">
-            <h2>contacta con nosotros</h2>
+            <h2>Contacta con nosotros</h2>
             <p>Déjanos tus datos y nos pondremos en contacto contigo</p>
         </div>
     </div>
     <div class="row justify-content-center">
 
         {% if page.id == 'distribuidores' %}
             <div class="col col-md-4 mb-4">
-                <a href="{{ site.form_distributor_url }}" target="_blank"
-                   class="btn btn-lg btn-frogtek btn-block">¡quiero distribuir Tiendatek!</a>
+                <a href="{{ site.form_distributor_url }}" onClick="gtag('event', 'distribuir', {'source' : 'distribuidores'})" 
```

Comment:
IMO, we should always have a simpler (less verbose) wrapper around Analytics calls, something like `trackEvent('distribuir', 'distribuidores');`, `trackEvent('lo quiero', 'contacto');`. With a good SoC, we don't rely of Google Analytics explicitly for tracking.

Reply:
@durey What do you think of the new change? Is that what you said?

Reply:
Exactly 💯